src/components/shareable-url-modal/shareable-url-modal.js

```diff
@@ -111,6 +167,18 @@ const ShareableUrlModal = ({ onToggleModal, visible }) => {
       if (request.ok) {
         setResponseUrl(response.url);
         setDeploymentState('success');
+        setToDisplayPublishedView();
```

Comment:
Can u pls explain the purpose of this?

Reply:
hey so this is to make sure if the request is okay, we should run the check to see if localStorage is existed then update it the local state accordingly. it's now called `displayPublishedView`